appinventor/components/src/com/google/appinventor/components/runtime/Reasoner.java

```diff
@@ -38,6 +38,7 @@
 import java.util.Collections;
 import java.util.Iterator;
 import java.util.List;
+import java.util.stream.Collectors;
```

Comment:
`Collectors` was implemented in Android 7.0. For maximal compatibility, we probably want to switch to using a `StringBuilder` instead.

Reply:
Actually, it looks like this import isn't used anywhere, so just remove it.